cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/ValueChangeParsingStrategy.java

```diff
@@ -61,12 +68,19 @@ public ValueChangeParsingStrategy(ChangeParser changeParser) {
                     .collect(Collectors.toList());
         }
 
-        return Arrays.asList(new CfObjectValueChanged(change.getAffectedObject().get(),
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
```

Comment:
Should be `info`.

Reply:
I would recommend to use debug logging here because this log message could occur very often and does not provide valuable information to the user